src/mixins/common-mixin.ts

```diff
@@ -93,20 +93,27 @@ export default class CommonMixin extends Vue {
     this[prop] = this[prop]?.toUpperCase()
   }
 
-  /** The appropriate edit label for corrections or alterations. */
+  /** The appropriate edit label for corrections, alterations or change filings. */
   get editLabel (): string {
     if (this.isCorrectionFiling) return 'Correct'
     if (this.isAlterationFiling || this.isChangeFiling) return 'Change'
     return 'Edit'
   }
 
-  /** The appropriate edited label for corrections or alterations. */
+  /** The appropriate edited label for corrections, alterations or change filings. */
   get editedLabel (): string {
     if (this.isCorrectionFiling) return 'Corrected'
     if (this.isAlterationFiling || this.isChangeFiling) return 'Changed'
     return 'Edited'
   }
 
+  /** The appropriate edits saved label for corrections, alterations or change filings. */
+  get editSavedLabel (): string {
+    if (this.isCorrectionFiling) return 'Corrections Saved'
+    if (this.isAlterationFiling || this.isChangeFiling) return 'Changes Saved'
+    return 'Edits Saved'
```

Comment:
A new display value for changes that take effect immediately i think

Reply:
It's not obvious when getters come from a mixin like this. Would it make sense to move these to the store getters file (or have a new getter file)? We already do this with some getters like isRoleStaff. Just thinking out loud. If this bothers noone else then nvm.

Reply:
My thoughts were the same as yours initially, it does feel a bit like a phantom computed value, especially if the component is extending a few mixins. 
That being said, it does cut down on the property decorators in the component itself and this is also the case for several of the computed labels/tags etc in the common mixin. 

So i guess i'm impartial, i'm fine with this because it's less code in our components and finding the computed values themselves are just a  Command (⌘) + click away. 🤷

Reply:
It doesn't bug me enough right now to change it. Maybe later 😁